docs/miscellaneous/create-dns-stamp.md

```diff
@@ -0,0 +1,79 @@
+# How to Create Your Own DNS Stamp for Secure DNS
+
+This guide will help you learn how to create your own DNS stamp for Secure DNS. Secure DNS is a service that enhances your internet security and privacy by encrypting your DNS queries. This prevents the possibility of your queries being intercepted or manipulated by malicious actors.
+
+For Secure DNS, `tls://`, `https://` or `quic://` URLs are normally used and it is enough for most users, and is recommended way.
+
+However, if you need additional security, like pre-resolved server IPs and certificate pinning by hash, you may generate your own DNS stamp. This guide tells you how to do it.
+
+## Introduction to DNS Stamps
+
+DNS Stamps are short strings that include all the information needed to connect to a secure DNS server. They simplify the process of setting up Secure DNS as the user does not need to manually enter all this data. 
```

Comment:
DNS stamps are short strings that contain all the information needed to connect to a secure DNS server.